packages/frontend/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-trybe-frontend",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

```suggestion
  "version": "1.3.0",
```

O script de publicação alterar a versão automaticamente. Não é necessário alterar aqui :) 
Se alterar manualmente, o script vai saltar o projeto para a `1.3.2`

---

Ah blza!